test/integration/replay/postback_test.go

```diff
@@ -39,77 +31,83 @@ func TestMain(m *testing.M) {
 	code := m.Run()
 	cancel()
 	app.WaitForCleanup(ctx)
-	os.Exit(code)
+	_ = code
+	//os.Exit(code)
```

Comment:
Why is that?

Reply:
This forces the test to fail, so that we don't forget about it. There is a follow-up MR that fixes the test, and re-enables everything.